src/sbi.rs

```diff
@@ -366,6 +382,11 @@ pub unsafe fn ecall_send(msg: &SbiMessage) -> Result<u64> {
 }
 
 #[cfg(not(all(target_arch = "riscv64", target_os = "none")))]
-unsafe fn ecall_send(_msg: &SbiMessage) -> Result<u64> {
+/// Test Compilation only.
```

Comment:
Was there any particular reason we couldn't wrap this in #cfg(test)?

Reply:
Wondered the same, but in general I think this is just a nice _contrapasso_ to the previous CFG for the `ecall_send`. :-)

Reply:
Ah, this isn't exactly test only. It is linked when I use this crate from a userspace VMM. The VMM can't make tee ecalls, it just makes syscalls. but it does need the rest of the crate, so panic is the right behavior in that case.